### PR TITLE
[MOB-2568] add in-app priority test for sorting on a list

### PR DESCRIPTION
**Pull request by roninopf**
## 🔹 JIRA Ticket(s) if any

* [MOB-2568](https://iterable.atlassian.net/browse/MOB-2568)

## ✏️ Description

* add another test for in-app priority

### CODE REVIEW

File: tests/swift-sdk-swift-tests/in-app-tests/InAppPriorityTests.swift [hunk 1]
```diff
@@ -13,6 +13,51 @@ class InAppPriorityTests: XCTestCase {
...
     
+    func testDisplayingPriorityLevelsInOrder() {
+        let condition1 = expectation(description: "not able to match message priority levels")
+        condition1.expectedFulfillmentCount = 4
+        
+        let messages = [
+            getMessageWithPriority("1", Const.PriorityLevel.low),
+            getMessageWithPriority("2", Const.PriorityLevel.high),
+            getMessageWithPriority("3", Const.PriorityLevel.medium),
+            getMessageWithPriority("4", Const.PriorityLevel.critical)
```
Comment (tapashmajumder) [inline tests/swift-sdk-swift-tests/in-app-tests/InAppPriorityTests.swift:24]: I forget why are priority levels stored in `Const`. I don't see `PriorityLevel.medium`, `PriorityLevel.critical` being used anywhere.

Comment (roninopf) [reply tests/swift-sdk-swift-tests/in-app-tests/InAppPriorityTests.swift:24]: I wasn't sure where else to put them (maybe in a dedicated `InAppPriority` Swift file?) but they're there if a developer wants to make comparisons (e.g. show this in-app if `Const.PriorityLevel.critical`)

Comment (tapashmajumder) [reply tests/swift-sdk-swift-tests/in-app-tests/InAppPriorityTests.swift:24]: I think these consts have internal visibility. And that is fine. I think all our Consts should be for internal consumption.

I think we should create a separate ticket for exposing and using PriorityLevels, if that is needed.

**Review by tapashmajumder — APPROVED** [review]
+1

**Pull request merged**